### PR TITLE
client update to 0.1.76

### PR DESCRIPTION
**Pull request by ekmineev**
Updated account domain authentication, inspector interface, desktop simulation interface to support the upgrade to version 0.1.76 of Remote Rendering client SDK.

### CODE REVIEW

File: ArrtModel/Model/ArrSessionManager.cpp [hunk 1]
```diff
@@ -904,7 +904,7 @@ void ArrSessionManager::startInspector()
         qInfo(LoggingCategory::renderingSession)
             << tr("Requesting connection to ARR inspector.\n")
             << m_lastProperties;
-        if (const auto async = m_session->ConnectToArrInspectorAsync(m_lastProperties.Hostname.c_str()))
```
Comment (mafranc) [inline ArrtModel/Model/ArrSessionManager.cpp:907]: how come this is not needed anymore?

Comment (ekmineev) [reply ArrtModel/Model/ArrSessionManager.cpp:907]: we made a change in client SDKs version 0.1.76 to remove the hostname parameter from this method as ConnectToArrInspectorAsync is session specific. Please see the release notes at https://github.com/Azure/azure-remote-rendering/releases/tag/2020-12-22

File: ArrtModel/ViewModel/ModelEditor/Viewport/ViewportModel.cpp [hunk 2]
```diff
@@ -374,10 +369,18 @@ void ViewportModel::updateProjection()
         ratio = (float)m_width / (float)m_height;
     }
     // update projection when viewport size changes
-    QMatrix4x4 m;
-    m.perspective(m_cameraSettings->getFovAngle(), ratio, m_cameraSettings->getNearPlane(), m_cameraSettings->getFarPlane());
+	QMatrix4x4 m;
+	m.perspective(m_cameraSettings->getFovAngle(), ratio, m_cameraSettings->getNearPlane(), m_cameraSettings->getFarPlane());
```
Comment (ChristopherManthei) [inline ArrtModel/ViewModel/ModelEditor/Viewport/ViewportModel.cpp:373]: Tabs vs spaces.

Comment (ekmineev) [reply ArrtModel/ViewModel/ModelEditor/Viewport/ViewportModel.cpp]: Fixed!

**Review by mafranc — COMMENTED** [review]
You shouldn't leave any real ID in the screenshot

**Comment by ekmineev** [comment]
> You shouldn't leave any real ID in the screenshot

Replaced with x's, although this id is not consequential at all - from an account I can delete after I post this.

**Review by mafranc — APPROVED** [review]

**Pull request merged**